cli/src/install.sh

```diff
@@ -131,6 +131,17 @@ copy_files() {
     (umask 077; mkdir -p "${data_dir}")
     cp -a "completions" "${data_dir}/"
     success "Copied completions to ${completions_dir}"
+
+    # Install extensions
+    for ext in extensions/*/PhylumExt.toml
+    do
+        ext=$(dirname "${ext}")
+
+        # Install the extension
+        # Note: We ignore the exit code here because the user might decline the install
+        "${bin_dir}/${bin_name}" extension install --accept-permissions "${ext}" || true
```

Comment:
I will update the note in #641 because that decision has been reversed. The conversation around this issue (which can be seen [here](https://github.com/phylum-dev/cli/pull/678#discussion_r971189213)) was the reason that the old PR (#678) was closed and issue #400 was worked before this one.

Unattended upgrades are still technically possible as can be seen in the following example:

```
❯ ./install.sh 2>&1 | tee output.txt

    phylum-cli installer

    OK Copied completions to /Users/kyle/.local/share/phylum/completions
Installing extension npm...
✅ Extension npm already installed, nothing to do
Installing extension poetry...
❗ Error: install aborted
Installing extension yarn...
✅ Extension yarn already installed, nothing to do
    OK Installed default extensions
    OK Completions are enabled for bash.
    OK Completions are enabled for zsh.
    OK Successfully installed phylum.

    Source your /Users/kyle/.zshrc file, add /Users/kyle/.local/bin to your $PATH variable, or
    log in to a new terminal in order to make `phylum` available.
```

The code notices that stdout is not a TTY and aborts the install of the `poetry` extension. I was not prompted and the script did not hang.

This is certainly not the most ideal situation, but I do believe it's reasonable without additional information about the user's intentions.

Reply:
Definitely not ideal. The thought was more along the lines of allowing an `--overwrite-extensions` flag to the install script and passing that user intention along to the `extension install` command (vs. simply hard-coding the `--overwrite` flag into the script).

It's not a big enough issue to stall this PR, but it is likely to come up as a desired feature at some point. In fact, it can already be imagined how waiting on user input will break the `phylum-ci` script entry point when using the `--phylum-release` option and `--force-install` flag.

Reply:
To be clear, this is not an issue for initial install because there should never be anything to overwrite. This is only a potential issue for upgrade. And I do not currently know of any automated upgrade use cases. And it is very hard to make decisions about what to do when the use case is unknown.